flatdata-py/flatdata/lib/errors.py

```diff
@@ -38,3 +38,55 @@ class MissingResourceError(KeyError, CorruptArchiveError):
     """
     def __init__(self, key):
         super().__init__("Resource {key} not found".format(key=key))
+
+
+class ResourceReadOnlyError(RuntimeError):
+    """
+    Resource is read only and cannot be written to.
+    """
+
+
+class MissingFieldError(RuntimeError):
+    def __init__(self, key, name):
```

Comment:
We probably should add descriptions to all errors

Reply:
done